code/modules/mob/living/living_movement.dm

```diff
@@ -2,6 +2,9 @@
 	. = ..()
 	update_turf_movespeed(loc)
 	update_looking_move()
+	//color correction
+	var/area/A = get_area(loc)
+	apply_color_correction(A?.color_correction)
```

Comment:
A bit worried about this potentially getting expensive

Reply:
Could employ COMSIG_MOVABLE_ENTERED_AREA instead?

Reply:
That's heaps better, yeah